internal/commands/scan_test.go

```diff
@@ -128,14 +128,18 @@ func TestScanCreate_ExistingApplicationAndProject_CreateProjectUnderApplicationS
 	execCmdNilAssertion(t, "scan", "create", "--project-name", "MOCK", "--application-name", "MOCK", "-s", dummyRepo, "-b", "dummy_branch")
 }
 
+func TestScanCreate_PassingImportFileTypeAndImportFilePath_CreateProjectAndImportSARIFFileSuccessfully(t *testing.T) {
+	execCmdNilAssertion(t, "scan", "create", "--project-name", "MOCK", "--import-file-path", "my-path", "--import-file-type", "my-type", "-s", dummyRepo, "-b", "dummy_branch")
```

Comment:
my-type should be SARIF ?

Reply:
not relevant anymore. we removed this flag